src/Widgets/AppEntry.vala

```diff
@@ -42,26 +52,55 @@ public class Notifications.AppEntry : Gtk.ListBoxRow {
             name = _("Other");
         }
 
+        var image = new Gtk.Image.from_icon_name ("pan-end-symbolic", SMALL_TOOLBAR);
+        image.get_style_context ().add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+
         var label = new Gtk.Label (name) {
             hexpand = true,
             xalign = 0
         };
         label.get_style_context ().add_class (Granite.STYLE_CLASS_H3_LABEL);
 
+        var expander_content = new Gtk.Box (HORIZONTAL, 3);
+        expander_content.add (image);
+        expander_content.add (label);
+
+        expander = new Gtk.ToggleButton () {
+            child = expander_content,
+            can_focus = false
+        };
+        unowned var expander_style_context = expander.get_style_context ();
+        expander_style_context.add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+        expander_style_context.add_class (Gtk.STYLE_CLASS_FLAT);
+        expander_style_context.add_class ("image-button");
+
         var clear_btn_entry = new Gtk.Button.from_icon_name ("edit-clear-all-symbolic", Gtk.IconSize.SMALL_TOOLBAR) {
             tooltip_text = _("Clear all %s notifications").printf (name)
         };
         clear_btn_entry.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
 
-        var grid = new Gtk.Grid () {
-            column_spacing = 6
-        };
-        grid.add (label);
-        grid.add (clear_btn_entry);
+        var box = new Gtk.Box (HORIZONTAL, 6);
+        box.add (expander);
+        box.add (new Gtk.Separator (VERTICAL));
```

Comment:
Instead of adding a separator here, we should switch the position of the label and the expander arrow:

![Screenshot from 2023-08-29 13 12 12](https://github.com/elementary/wingpanel-indicator-notifications/assets/7277719/aaffc064-c82a-4925-b2dd-98831792a16e)